index.html

```diff
@@ -149,6 +146,54 @@ <h2>
   ]
 }
         </pre>
+        
+        <pre class="example" title="Example did:web DID document using JWS2020">
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
+  ],
+  "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
```

Comment:
```suggestion
  "id": "did:key:web:example.com#owner",
```

Reply:
^ nope... `id` MUST be a DID, NOT a DID URL!

Reply:
This should be ` "id": "did:web:example.com",`